framework/filters/Cors.php

```diff
@@ -80,6 +80,10 @@ class Cors extends ActionFilter
      */
     public $actions = [];
     /**
+     * @var bool handle the preflight check and don't proceed with the execution of the request.
+     */
+    public $handleOptions = true;
```

Comment:
It also does make sure a 200 OK is sent, that's for the handling part.

What about `acceptOptions`? That would suggest that if you disable it, the function won't accept the options, what it does.